homeassistant/components/tis_control/config_flow.py

```diff
@@ -0,0 +1,70 @@
+"""Config flow for TISControl integration."""
+
+from __future__ import annotations
+
+import logging
+
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigFlow, ConfigFlowResult
+from homeassistant.const import CONF_PORT
+from homeassistant.core import callback
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema({vol.Required(CONF_PORT): str}, required=True)
```

Comment:
Why is the port validated as a str?

Reply:
sorry for the confusion i thought the form returns only strings